examples/hello_part_no_macros/src/main.rs

```diff
@@ -0,0 +1,153 @@
+//! # Example `hello_part_no_macros`
+//!
+//! ## FAQ
+//!
+//! - What is [`StartContext<Hypervisor>`]?
+//!   - Some parts of the ARINC 653 API are only available on a specific mode.
+//!     In particular, the process and port creationg API can only be called
+//!     during {`COLD`,`WARM`}`_START` mode (see [`OperatingMode`]).
+//!     `StartContext` is a struct that exposes this API only during
+//!     {`COLD`,`WARM`}`_START` mode, and can not be constructed by the user.
+//!     This makes misuse of the API (as in attempting to create a port/process
+//!     during `NORMAL`) impossible.
```

Comment:
This should be part of the crate-level documentation and link to it.

Reply:
Yes, actually, this should be part of the `a653rs` documentation.